fdbclient/NativeAPI.actor.cpp

```diff
@@ -2710,22 +2710,31 @@ TransactionOptions::TransactionOptions(Database const& cx) {
 	}
 }
 
-TransactionOptions::TransactionOptions() {
-	memset(this, 0, sizeof(*this));
+void TransactionOptions::clear() {
```

Comment:
Can you add a comment to `TransactionOptions` saying something like "if adding a new field, remember to update TransactionOptions::clear" ?